config/rbac/byohost_editor_role.yaml

```diff
@@ -24,3 +24,14 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
```

Comment:
Is it possible to narrow down the RBAC for events only for `ByoHost` CRDs?

Reply:
done